code/modules/surgery/organs/lungs.dm

```diff
@@ -106,11 +107,25 @@
 
 	//Too much oxygen! //Yes, some species may not like it.
 	if(safe_oxygen_max)
-		if(O2_pp > safe_oxygen_max)
+		if((O2_pp > safe_oxygen_max) && safe_oxygen_max ==0)
```

Comment:
This check will never pass.

Reply:
I ended up removing these checks "&& safe_oxygen_max ==0)" as every race that wasnt plasmamen it worked with and plasmamen already had a hard time with oxygen and didnt need to take more suffocation damage